src/navigator/DrawerNavigator.tsx

```diff
@@ -310,16 +308,14 @@ export default function DrawerNavigator({ route }: Props) {
             // @ts-expect-error component type in native-stack v6
             component={WalletSecurityPrimer}
             options={{
-              drawerLabel: !backupCompleted
```

Comment:
this condition is no longer required because this item will only show if backup isn't completed